desktop-app/app/main.dev.js

```diff
@@ -90,6 +90,9 @@ const openUrl = url => {
  */
 
 app.on('will-finish-launching', () => {
+  if (process.platform === 'win32') {
+    urlToOpen = process.argv.filter(i => /^responsively/.test(i))[0];
+  }
```

Comment:
here I am checking if it's `Windows`, we are going to assign the `url` to `urlToOpen`.